node/grpc/server_v2.go

```diff
@@ -367,3 +380,62 @@ func (s *ServerV2) GetChunks(ctx context.Context, in *pb.GetChunksRequest) (*pb.
 		ChunkEncodingFormat: pb.ChunkEncodingFormat_GNARK,
 	}, nil
 }
+
+// validateDispersalRequest validates the DisperseBlobRequest and returns the blob header
+// Differences between this and the DispersalServerV2 are:
+// - Takes *corev2.BlobCertificate instead of DisperseBlobRequest
+// - no encoding prover GetCommitmentsForPaddedLength check
+// - directly take blob lengths (no blob data yet)
+// - doesn't check every 32 bytes is a valid field element
```

Comment:
Is it useful to check these items at validator here as well?

Reply:
It would be, but not possible, because those checks are done to the blob directly while we shouldn't make validators validate request against the blob itself

Reply:
Where is it checked?
It's generally better to check at earlier stage if there's sufficient information so we can reject invalid requests fast and cheaply.

Reply:
It isn't checked by the validators. I don't think it is possible because it requires the blob data itself:(